draft-kazuho-httpbis-priority.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: The Priority HTTP Header Field
+title: Extensible Prioritization for HTTP
```

Comment:
Should this rather be "Extensible Prioritization Scheme for HTTP"?

Reply:
I think that sounds more true to the nature of the document

Reply:
Thanks! Applied in aef4574.